components/camel-mllp/src/main/java/org/apache/camel/component/mllp/MllpTcpClientProducer.java

```diff
@@ -193,7 +194,14 @@ public synchronized void process(Exchange exchange) throws MllpException {
                     mllpBuffer.resetSocket(socket);
                 }
             }
-
+            if (getConfiguration().getExchangePattern() == ExchangePattern.InOnly) {
+                log.debug("process({}) [{}] - not checking acknowledgement from external system",
+                        exchange.getExchangeId(), socket);
+                if (exchange.getException() != null) {
+                    exchange.getException().printStackTrace();
```

Comment:
You should not print the exception, this should be removed

Reply:
Sorry, completely forgot to remove this.